### PR TITLE
[WFCORE-2806] Add the ability to resolve standard server system prope…

### PR DESCRIPTION
**Pull request by yersan**
…rties to configure the managed server JVMs

Jira issue: https://issues.redhat.com/browse/WFCORE-2806

### CODE REVIEW

File: core/WFCORE-2806_Enable_resolution_of_standard_server_system_properties_on_domain_mode.adoc [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* mailto:jjedlick@redhat.com[Jakub Jedlicka]
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE.
+// Discuss with QE during the Kickoff state to decide this
+* [X] Engineering
```
Comment (jedla97) [inline core/WFCORE-2806_Enable_resolution_of_standard_server_system_properties_on_domain_mode.adoc:68]: Can you change it to `* [x] Engineering`. It's create this sign ✓.

**Review by jedla97 — COMMENTED** [review]
It's one small change otherwise LGTM from QA point of view.

File: core/WFCORE-2806_Enable_resolution_of_standard_server_system_properties_on_domain_mode.adoc (hunk 1 above)
Comment (yersan) [reply core/WFCORE-2806_Enable_resolution_of_standard_server_system_properties_on_domain_mode.adoc]: Fixed, thanks!

**Review by jedla97 — APPROVED** [review]
How I say earlier LGTM from QE point of view

File: core/WFCORE-2806_Enable_resolution_of_standard_server_system_properties_on_domain_mode.adoc [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+== Requirements
+
+=== Hard Requirements
+
+* The following standard server system properties will be available as management model expressions to configure the managed server JVM:
+** jboss.server.base.dir: Root directory for an individual server instance.
+** jboss.server.data.dir: Directory the server will use for persistent data file storage.
+** jboss.server.log.dir: Directory the server will use for log file storage.
+** jboss.server.temp.dir: Directory the server will use for temporary file storage.
+* These environment variables will be available to configure the jvm resources at:
```
Comment (bstansberry) [inline core/WFCORE-2806_Enable_resolution_of_standard_server_system_properties_on_domain_mode.adoc:100]: I'd say

"The use of these properties to resolve expressions will be available for attributes in the jvm resources at:"

**Comment by bstansberry** [comment]
Minor point about wording aside, this LGTM.

File: core/WFCORE-2806_Enable_resolution_of_standard_server_system_properties_on_domain_mode.adoc (hunk 2 above)
Comment (yersan) [reply core/WFCORE-2806_Enable_resolution_of_standard_server_system_properties_on_domain_mode.adoc]: Done. Thanks for reviewing it.

**Review by bstansberry — APPROVED** [review]

**Pull request merged**